.ci/config/matrices.yaml

```diff
@@ -7,7 +7,7 @@ engine-webapp-unit:
   databases:
   - 'postgresql_96'
   - 'postgresql_94'
-  - 'postgresql_107'
+  - 'postgresql_1013'
```

Comment:
The database label should be updated in the `daily-stages` as well (on line 50).